temba/ussd/models.py

```diff
@@ -37,7 +37,8 @@ def start_session_async(self, flow, urn, content, date, message_id):
         message = Msg.objects.create(
             channel=self.channel, contact=self.contact, contact_urn=self.contact_urn,
             sent_on=date, session=self, msg_type=USSD, external_id=message_id,
-            created_on=timezone.now(), modified_on=timezone.now(), org=self.channel.org)
+            created_on=timezone.now(), modified_on=timezone.now(), org=self.channel.org,
+            direction=self.USSD_PULL)
```

Comment:
Using `USSD_PULL` here is slightly weird to me, only because it REALLY matters that we only use `I` and `O` for directions because of how indexes are build and queries are optimized and my concern is that we could change the constants above to be something else and not notice that we've screwed ourselves in that respect.

How do you feel about just using `Msg.INCOMING`?